index.js

```diff
@@ -48,8 +48,27 @@ const supportsAbortController = typeof globals.AbortController === 'function';
 const supportsStreams = typeof globals.ReadableStream === 'function';
 const supportsFormData = typeof globals.FormData === 'function';
 
+const mergeHeaders = (source1, source2) => {
```

Comment:
```suggestion
const mergeHeaders = (...args) => {
```

Reply:
That's exactly the opposite of what @sholladay asked me to do.

Reply:
Sorry for the conflicting requests. My stance on this is that it's basically a reducer function, should take two arguments only (existing headers and override headers), and return the computed headers. To do that, I'm pretty sure we only need to coerce the first argument to a `Headers` instance and then loop over the second argument to override the header values. I don't think we need to loop over the list of arguments. But I could be wrong.

If it's true that we actually do need to loop over the arguments list, then I am fine with using spread syntax here, as it's pretty equivalent to making the array ourselves..

Reply:
You are correct. I initially had it this way, I don't recall why It was changed to this form. Thanks! @sholladay

Reply:
I'm fine with either :)